worker/uniter/runner/debug/server.go

```diff
@@ -102,8 +102,14 @@ export | grep -v $FILTER > $JUJU_DEBUG/env.sh
 cat > $JUJU_DEBUG/welcome.msg <<END
 This is a Juju debug-hooks tmux session. Remember:
 1. You need to execute hooks manually if you want them to run for trapped events.
-2. When you are finished with an event, you can run 'exit' to close the current window and allow Juju to continue running.
-3. CTRL+a is tmux prefix.
+2. When you are finished with an event, you can run 'exit' to close the current window and allow Juju to continue processing
+new events for this unit without exiting a current debug-session.
+3. To run a hook, end a debugging session and avoid processing any more events manually use
```

Comment:
To run a hook and end the debugging session avoiding processing any more events manually, use:


?